### PR TITLE
Deprecation: Issue 6227 - Comparison of different enums

### PR DESCRIPTION
**Pull request by wilzbach**
From https://github.com/dlang/dmd/pull/6821

CC @UplinkCoder

### CODE REVIEW

**Comment by UplinkCoder** [comment]
looks good.

**Comment by wilzbach** [comment]
The deprecation is already part of DMD, so anything blocking this?

**Comment by wilzbach** [comment]
CC @CyberShadow - can we get this shipped before the 2.075 release which includes this deprecation? :)

File: deprecate.dd [hunk 1]
```diff
@@ -57,6 +58,41 @@ $(SPEC_S Deprecated Features,
         $(DD The feature is completely gone)
     )
 
+$(H3 $(DEPNAME Implicit comparison of different enums))
+    $(P Currently, comparison of different enumerated type is allowed:
```
Comment (CyberShadow) [inline deprecate.dd]: "Currently" is not going to be current for much longer

**Comment by wilzbach** [comment]
> "Currently" is not going to be current for much longer

Hehe, I copied this from the entry below -> I changed the tense for both entries.

**Pull request merged**